mingw-w64-emacs/PKGBUILD

```
@@ -69,6 +65,10 @@ build() {
   # --without-compress-install is needed because we don't have gzip in
   # the mingw binaries.
 
+  # Workaround: make fails with bootstrap-emacs.pdmp and emacs.pdmp
+  # for unknown reason.
+  make || true
+  make || true
   make
```

```suggestion
  make || make || make
```

??  I've had to do things like that before ☹️